.github/workflows/render-stan-docs.yaml

```diff
@@ -0,0 +1,33 @@
+# Based on https://github.com/mattnotmitt/doxygen-action
+name: Doxygen Action
+
+on:
+  pull_request:
```

Comment:
We likely only want to deploy on main so may want a different trigger for that (still build on PRs but not try and publish and just push to an environment variable).

Reply:
Resolved through #502 by @seabbs.